wagtail/wagtailforms/models.py

```diff
@@ -295,7 +295,7 @@ def process_form_submission(self, form):
 
     def send_mail(self, form):
         addresses = [x.strip() for x in self.to_address.split(',')]
-        content = '\n'.join([x[1].label + ': ' + text_type(form.data.get(x[0])) for x in form.fields.items()])
+        content = '\n'.join([x[1].label + ': ' + text_type( ', '.join(dict(form.data.lists())[x[0]]) ) for x in form.fields.items()])
```

Comment:
You're right, I still get one failure with that code, but that is fixed by converting an empty list to a string of 'None', like so:

```python
        content = []
        for field in form:
            value = field.value()
            if isinstance(value, list):
                value = ', '.join(value)
                if len(value) == 0:
                    value = 'None'
            content.append('{}: {}'.format(field.label, value))
        content = '\n'.join(content)
```